app/views/shared/_tax_return_list.html.erb

```diff
@@ -45,7 +45,8 @@
           <% end %>
         <% else %>
           <div>
-            <div class="tax-return-list__status label label--status"><%= stage_translation_from_status(tax_return.status) %>/<%= status_translation(tax_return.status) %></div>
+            <% status_text = "#{stage_translation_from_status(tax_return.status)}/#{status_translation(tax_return.status)}" %>
+            <div class="tax-return-list__status label label--status" data-status="<%= status_text %>"><%= status_text %></div>
```

Comment:
set data-status to tax_return.status, and then you wont have to worry about spacing or translations in the css, which I think will make things less brittle. For example, say we decide to change how we display the label: put it on two lines, remove the spaces between the back slashes. We'd have to update the css or the label colors would break. If we use the raw status, we only have to update if we change the enumerable for the status (which is far less likely). the raw status includes both the stage and status